docs/sources/configuration/_index.md

```diff
@@ -2278,6 +2290,41 @@ The `tracing_config` block configures tracing for Jaeger. Currently limited to d
 [enabled: <boolean>: default = true]
 ```
 
+## common_config
+
+The `common_config` block sets common definitions to be shared by different components.
+This way, one doesn't have to replicate configs in multiple places.
+
+```yaml
+# A common storage config to be used by the different Loki components.
+[storage: <common_storage_config>]
+
+# When defined, the given prefix will be present in front of the endpoint paths.
```

Comment:
This sentence is misleading. This `path_prefix` is about internal directories not URLs. See config_wrapper.go [here](https://github.com/grafana/loki/blob/v2.4.2/pkg/loki/config_wrapper.go#L252-L279).

Reply:
Yes it is wrong, I created [this issue](https://github.com/grafana/loki/issues/5119) after this PR got merged.